samples/matter/common/src/persistent_storage/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if NCS_SAMPLE_MATTER_PERSISTENT_STORAGE
+
+config NCS_SAMPLE_MATTER_SETTINGS_STORAGE_BACKEND
+	bool "Settings based storage implementation for Matter samples"
+	default y
+
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
+	bool "Secure storage implementation for Matter samples"
+	select TRUSTED_STORAGE
+	select PSA_PROTECTED_STORAGE
+	default n
+
+if NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
+
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_MAX_ENTRY_NUMBER
+	int "Maximum number of entries that can be stored securely"
+	default 40
+
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_PSA_KEY_VALUE_OFFSET
+	hex "Minimal PSA key allowed for Matter samples"
```

Comment:
Hmm, maybe "Starting number" instead of "Minimal"? Word Minimal may be misleading here.

Reply:
Ok, maybe "The PSA key offset dedicated for Matter samples"?